src/os/bluestore/BlueFS.cc

```diff
@@ -3594,3 +3594,10 @@ void OriginalVolumeSelector::dump(ostream& sout) {
     << ", slow_total:" << slow_total
     << std::endl;
 }
+
+// ===============================================
+// WithoutSlowDirVolumeSelector
```

Comment:
@Huehueh - mind rename this class to something like FitToFastVolumeSelector?

Reply:
Ok, no problem!